packages/components/src/Input/Input.spec.js

```diff
@@ -0,0 +1,61 @@
+import { getByTestId, render } from '@testing-library/react';
+import { ThemeProvider, createTheme } from '@platzily-ui/styling';
+import Input from './Input';
+
+describe('@Components/Input', () => {
+  it('Given the Input Component, when the props provide an object within the styles attribute then the component will take those styles ', () => {
+    // arrange
+    const theme = createTheme();
+    const { getByRole } = render(
+      <ThemeProvider theme={theme}>
+        <Input role="banner" style={{ color: theme.palette.secondary.main }} />
+      </ThemeProvider>,
+    );
+
+    // act
+    const InputTestedText = getByRole('banner');
```

Comment:
This can be done using another role, like textbox, the inputs have that role by default. And the banner role won't be needed.